### PR TITLE
Handle signals from jail

### PR DESCRIPTION
**Pull request by rasom**
Utilizes https://github.com/status-im/status-go/pull/310 and allows to define another signal for #1810 without making changes in `status-go`.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/events.cljs [hunk 1]
```diff
@@ -210,6 +210,20 @@
  (fn [_ _]
    {::get-fcm-token-fx nil}))
 
+(defn handle-jail-signal [{:keys[chat_id data]}]
+  (println :DAAAATA data)
```
Comment (janherich) [inline src/status_im/ui/screens/events.cljs]: We should probably remove that

File: src/status_im/ui/screens/events.cljs [hunk 2]
```diff
@@ -210,6 +210,20 @@
...
+  (let [{:keys [event data]} (types/json->clj data)]
+    (println :DAAAATA2 data (string? data))
```
Comment (janherich) [inline src/status_im/ui/screens/events.cljs]: Same as above

File: src/status_im/ui/screens/events.cljs [hunk 3]
```diff
@@ -224,8 +238,7 @@
         "local_storage.set"       (dispatch [:set-local-storage event])
```
Comment (janherich) [inline src/status_im/ui/screens/events.cljs]: Why is this `local_storage` signal handled in two places, here and in `case` statement of the `handler-jail-signal` function ? Or are those two different events ?

File: src/status_im/ui/screens/events.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/events.cljs]: sure :D

Comment (rasom) [reply src/status_im/ui/screens/events.cljs]: done

File: src/status_im/ui/screens/events.cljs (hunk 3 above)
Comment (rasom) [reply src/status_im/ui/screens/events.cljs]: that's leftover, removed, thanks

**Review by janherich — APPROVED** [review]

File: src/status_im/ui/screens/events.cljs [hunk 4]
```diff
@@ -210,6 +210,18 @@
  (fn [_ _]
    {::get-fcm-token-fx nil}))
 
+(defn handle-jail-signal [{:keys[chat_id data]}]
```
Comment (oskarth) [inline src/status_im/ui/screens/events.cljs:213]: space in keys

File: src/status_im/ui/screens/events.cljs [hunk 5]
```diff
@@ -210,6 +210,18 @@
...
+  (let [{:keys [event data]} (types/json->clj data)]
+    (case event
+      "local-storage" (dispatch [:set-local-storage {:chat-id chat_id
+                                                     :data    data}])
+      "show-suggestions" (dispatch [:show-suggestions-from-jail {:chat-id chat_id
+                                                                 :markup  data}])
+      "send-message" (dispatch [:send-message-from-jail {:chat-id chat_id
+                                                         :message data}])
+
+      (log/debug "Unknown jail signal " type))))
```
Comment (oskarth) [inline src/status_im/ui/screens/events.cljs:223]: type type :)

**Pull request merged**